From zbr@ioremap.net  Wed Jan 21 09:08:11 2009
From: Evgeniy Polyakov <zbr@ioremap.net>
Date: Mon, 19 Jan 2009 21:30:47 +0300
Subject: Staging: DST: fix build dependancy
To: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Stephen Rothwell <sfr@canb.auug.org.au>, Greg KH <gregkh@suse.de>
Message-ID: <20090119183046.GA28947@ioremap.net>
Content-Disposition: inline


On Mon, Jan 19, 2009 at 10:12:24AM -0800, Randy Dunlap (randy.dunlap@oracle.com) wrote:
> 
> DST build fails when CONFIG_BLOCK=n:

DST should depend on block and block device, in the original patch its
kconfig entry was in the BLK_DEV menu, so this dependency was satisfied
automatically. Should attached patch be pushed into drivers/staging?

Reported-by: Randy Dunlap <randy.dunlap@oracle.com>
Acked-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Evgeniy Polyakov <zbr@ioremap.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/dst/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/dst/Kconfig
+++ b/drivers/staging/dst/Kconfig
@@ -1,6 +1,6 @@
 config DST
 	tristate "Distributed storage"
-	depends on NET && CRYPTO && SYSFS
+	depends on NET && CRYPTO && SYSFS && BLK_DEV
 	select CONNECTOR
 	---help---
 	DST is a network block device storage, which can be used to organize
